### PR TITLE
NAS-118891 / 22.12 / Used snapshot size not showed on the storage page (additional)

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/interfaces/dataset.interface.ts [hunk 1]
```diff
@@ -36,7 +38,6 @@ export interface Dataset {
   usedbychildren: ZfsProperty<number>;
   usedbydataset: ZfsProperty<number>;
   usedbyrefreservation: ZfsProperty<number>;
-  usedbysnapshots: ZfsProperty<number>;
```
Comment (denysbutenko) [inline src/app/interfaces/dataset.interface.ts:39]: Is it been removed from the API response?

File: src/app/pages/datasets/components/dataset-capacity-management-card/space-management-chart/space-management-chart.component.ts [hunk 2]
```diff
@@ -57,16 +57,17 @@ export class SpaceManagementChartComponent implements OnChanges {
   }
 
   private updateChartData(): void {
+    // TODO: Used by Snapshots was removed.
```
Comment (denysbutenko) [inline src/app/pages/datasets/components/dataset-capacity-management-card/space-management-chart/space-management-chart.component.ts:60]: What means TODO?

**Review by denysbutenko — APPROVED** [review]
Let's keep interfaces untouched otherwise looks good.

File: src/app/pages/datasets/components/dataset-capacity-management-card/space-management-chart/space-management-chart.component.ts (hunk 2 above)
Comment (AlexKarpov98) [reply src/app/pages/datasets/components/dataset-capacity-management-card/space-management-chart/space-management-chart.component.ts:60]: @denysbutenko - it's a reminder that some data was removed

File: src/app/interfaces/dataset.interface.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/interfaces/dataset.interface.ts:39]: @denysbutenko - I don't think so, but we will not use it on the FE side, do you think we should keep it?

Comment (denysbutenko) [reply src/app/interfaces/dataset.interface.ts:39]: yes please

File: src/app/interfaces/dataset.interface.ts [hunk 3]
```diff
@@ -179,7 +181,6 @@ export interface DatasetDetails {
   used: ZfsProperty<number>;
   usedbychildren: ZfsProperty<number>;
   usedbydataset: ZfsProperty<number>;
-  usedbysnapshots: ZfsProperty<number>;
```
Comment (denysbutenko) [inline src/app/interfaces/dataset.interface.ts:182]: Dataset details still have `usedbysnapshots` property also

**Review by denysbutenko — CHANGES_REQUESTED** [review]

**Review by denysbutenko — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**